app/code/Magento/Eav/Model/Entity/Attribute/AbstractAttribute.php

```diff
@@ -636,7 +636,11 @@ public function getSource()
             } else {
                 $this->_source = $this->getSourceModel();
             }
-            $source = $this->_universalFactory->create($this->_source);
+            try {
+                $source = $this->_universalFactory->create($this->_source);
+                } catch (\ReflectionException $e) {
+                $source = false;
```

Comment:
Instead of using `false`, consider returning `null` to indicate the absence of a valid source object.